odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -524,6 +524,12 @@ def test_23_relation(self):
         self.assertEqual(message.env, self.env)
         self.assertEqual(message.discussion.env, self.env)
 
+        # Check for description value can_create, can_write and can_unlink for relation field
+        field = message._fields['author'].get_description(self.env)
+        self.assertEqual(field.get('can_create'), not None)
+        self.assertEqual(field.get('can_write'), not None)
+        self.assertEqual(field.get('can_unlink'), not None)
```

Comment:
Be careful: `not None` is the value `True`. That's not what you mean.
Simply check that field contains the three given keys: `self.assertIn('can_create', field)`...